### PR TITLE
user/-j-andrew/test&automation-edit

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
AB#43161395

### CODE REVIEW

File: microsoft-edge/test-and-automation/test-and-automation.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ There are many tools to automate your testing of Microsoft Edge:
 | Playwright | The Playwright library provides cross-browser automation through a single API. |
 | Puppeteer | The Puppeteer library provides a high-level API to control browsers, including Microsoft Edge, using the DevTools Protocol. |
 | WebDriver | Automates testing in Microsoft Edge by simulating user interaction.  Provides advantages over JavaScript unit tests. |
-| webhint | Checks your code for best practices and common errors, to test and improve accessibility, performance, cross-browser and PWA compatibility, and security of your site. |
+| Webhint | Checks your code for best practices and common errors, to test and improve accessibility, performance, cross-browser and PWA compatibility, and security of your site. |
```
Comment (zoherghadyali) [inline microsoft-edge/test-and-automation/test-and-automation.md:22]:
```suggestion
| webhint | Checks your code for best practices and common errors, to test and improve accessibility, performance, cross-browser and PWA compatibility, and security of your site. |
```
webhint is intentionally not capitalized. See https://webhint.io/

File: microsoft-edge/origin-trials/index.md [hunk 2]
```diff
@@ -5,13 +5,15 @@ author: MSEdgeTeam
...
+
 You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
 
-For more information, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials).  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
+See the [Microsft Edge Origin Trials Developer Console](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/) to see a list of the available origin trials and register your origin to give them a try.  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```
Comment (zoherghadyali) [inline microsoft-edge/origin-trials/index.md:16]:
```suggestion
See the [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/) to see a list of the available origin trials and register your origin to give them a try.  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```
This had a typo

File: microsoft-edge/origin-trials/index.md [hunk 3]
```diff
@@ -5,13 +5,15 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 01/07/2021
+ms.date: 02/02/2023
 ---
 # Use Origin Trials in Microsoft Edge
 
+Web content's origin is defined by the scheme (protocol), hostname (domain), and port of the URL used to access it. Two objects have the same origin only when the scheme, hostname, and port all match.  This hub,  Origin Trials provides space and access to test new origins.
```
Comment (zoherghadyali) [inline microsoft-edge/origin-trials/index.md:12]:
```suggestion
```
I don't think we need this content. We can cut this sentence.

File: microsoft-edge/origin-trials/index.md [hunk 4]
```diff
@@ -5,13 +5,15 @@ author: MSEdgeTeam
...
+
 You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
```
Comment (zoherghadyali) [inline microsoft-edge/origin-trials/index.md:14]:
```suggestion
You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.

Instead, you register the origin (for example, `https://www.example.com:443`) that you want to use the experimental feature on and you'll be issued a token that you add to the code of your website.
```

**Review by zoherghadyali — CHANGES_REQUESTED** [review]

File: microsoft-edge/origin-trials/index.md [hunk 5]
```diff
@@ -5,13 +5,16 @@ author: MSEdgeTeam
...
 
-For more information, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials).  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
+Instead, you register the origin (for example, `https://www.example.com:443`) that you want to use the experimental feature on and you'll be issued a token that you add to the code of your website.
+
+See the [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/) to see a list of the available origin trials and register your origin to give them a try.  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:17]:
```suggestion
To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/).  You can also provide feedback to browser engineers and the web standard community about the design of the API, your use cases, or your experience using the APIs .
```
* Used pattern "To do X, see Y."
* Removed en-us/
* Pulled end of sentence 2 toward the start.  Up to you which sequence of phrases, here's an experiment.

https://developer.microsoft.com/microsoft-edge/origin-trials/

File: microsoft-edge/test-and-automation/devtools-protocol.md [hunk 6]
```diff
@@ -5,10 +5,10 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/19/2021
+ms.date: 1/30/2023
 ---
 # DevTools Protocol
 
-Use the DevTools Protocol to instrument, inspect, debug, and profile browsers including Microsoft Edge.  The Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
+Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge.  By [building Microsoft Edge on the Chromium open-source project](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md), the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
```
Comment (mikehoffms) [inline microsoft-edge/test-and-automation/devtools-protocol.md:12]:
```suggestion
Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge.  By building Microsoft Edge on the Chromium open-source project, the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.  For information about the Chromium open-source project, see [Microsoft Edge and Chromium Open Source: Our Intent](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md).
```
Standard link pattern to show the destination text as link text. Corp style says do that if possible.

File: microsoft-edge/test-and-automation/test-and-automation.md [hunk 7]
```diff
@@ -27,7 +27,7 @@ These tools are described below.
 <!-- ====================================================================== -->
 ## DevTools Protocol
 
-Use the DevTools Protocol to instrument, inspect, debug, and profile browsers including Microsoft Edge.  The Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
+Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge. By [building Microsoft Edge on the Chromium open-source project](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md), the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
```
Comment (mikehoffms) [inline microsoft-edge/test-and-automation/test-and-automation.md:30]:
```suggestion
Use the DevTools Protocol to instrument, inspect, debug, and profile browsers, including Microsoft Edge.  By building Microsoft Edge on the Chromium open-source project, the Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.  For information about the Chromium open-source project, see [Microsoft Edge and Chromium Open Source: Our Intent](https://github.com/MicrosoftEdge/MSEdge/blob/master/README.md).
```
instance 2 of the above comment

File: microsoft-edge/test-and-automation/test-and-automation.md [hunk 8]
```diff
@@ -37,8 +37,7 @@ See [DevTools Protocol](devtools-protocol.md).
 
 You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
 
-See [Use Origin Trials in Microsoft Edge](../origin-trials/index.md).
-
+See the [Microsft Edge Origin Trials Developer Console](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/) to see a list of the available origin trials and register your origin to give them a try.  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```
Comment (mikehoffms) [inline microsoft-edge/test-and-automation/test-and-automation.md:40]:
```suggestion
To see a list of the available origin trials and register your origin to give them a try, see the [Microsft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/).  You can also provide feedback to browser engineers and the web standard community about the design of the API, your use cases, or your experience using the APIs.
```

File: microsoft-edge/origin-trials/index.md [hunk 9]
```diff
@@ -5,13 +5,16 @@ author: MSEdgeTeam
...
-ms.date: 01/07/2021
+ms.date: 02/02/2023
 ---
 # Use Origin Trials in Microsoft Edge
 
+
 You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
 
-For more information, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials).  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
+Instead, you register the origin (for example, `https://www.example.com:443`) that you want to use the experimental feature on and you'll be issued a token that you add to the code of your website.
```
Comment (mikehoffms) [inline microsoft-edge/origin-trials/index.md:15]:
```suggestion
Instead, you register the origin (for example, `https://www.contoso.com:443`) that you want to use the experimental feature on and you'll be issued a token that you add to the code of your website.
```

**Review by zoherghadyali — APPROVED** [review]

**Pull request merged**